fdbclient/CoordinationInterface.h

```diff
@@ -32,12 +32,15 @@ const int MAX_CLUSTER_FILE_BYTES = 60000;
 
 constexpr UID WLTOKEN_CLIENTLEADERREG_GETLEADER(-1, 2);
 constexpr UID WLTOKEN_CLIENTLEADERREG_OPENDATABASE(-1, 3);
+constexpr UID WLTOKEN_CLIENTLEADERREG_DESCRIPTOR_MUTABLE(-1, 4);
 
-constexpr UID WLTOKEN_PROTOCOL_INFO(-1, 10);
```

Comment:
Won't this break backward compatibility?

Reply:
I think probably this doesn't in practice because this endpoint was introduced on master, but I agree we need to be careful about changing this. We should probably add a comment that this endpoint is supposed to be a stable one and is not allowed to change.

Reply:
Now that we've cut the release-7.0 branch and this is going into master, I'm feeling less comfortable with changing this. Would it be ok to return it to 10 and adjust the other affected endpoints accordingly?

Reply:
I was advised in discussions with @etschannen that a new client endpoint had to be sequential with the rest of the client endpoints, so we need to insert the new one in the client list and then the others can bumped by 1 .. and the total number of endpoints becomes 11. I'll ping you offline about this.